### PR TITLE
Theme colors page

### PR DESCRIPTION
**Pull request by jordandrako**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Remake old theme colors page with the new color palette.

**Old:**
https://developer.microsoft.com/en-us/fabric#/styles/colors?fabricVer=5

**New:**
In the PR demo site: http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/9369/merge/fabric-website/dist/index.html?devhost#/styles/web/colors/theme-slots

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9369)

### CODE REVIEW

File: apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsOverview.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file contains the Overview section for the page. It should summarize the content without duplicating it.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsOverview.md]: Delete this?

Comment (jordandrako) [reply apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsOverview.md]: WIP, updating it now.

File: apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Colors can be applied to text, backgrounds, or borders using the following class name conventions:
```
Comment (Vitalius1) [inline apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md]: nit: Feels like this implementation section needs a revamp of some sort. Currently it only shows you the Fabric Core application. Should we also include a css-in-js story? Or I missed it and we have it somewhere else on the website?

File: apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+| Type                    | Fabric Core className                |
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md]: Very small nit: className should probably be just class

File: apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md (hunk 2 above)
Comment (Jahnp) [reply apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md]: Agreed with @Vitalius1 's comments, this page could probably connect with the [Theming wiki page](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Theming). We should probably discuss if it'd make sense to move some of that content here or keep it separate. I don't think we should block this PR on that, but would be curious to hear @phkuo and @ecraig12345 's thoughts.

**Comment by jordandrako** [comment]
@Vitalius1 @Jahnp I expanded the overview and implementation with some content from our styling and theming wiki articles. Can you review that for me?

File: apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md (hunk 2 above)
Comment (Jahnp) [reply apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md]: @jordandrako ok, these improvements look great!

**Comment by leddie24** [comment]
@ecraig12345 @jordandrako is this ready to be merged?

**Comment by jordandrako** [comment]
@ecraig12345 Thanks for merging this yesterday. I was OOF.

**Pull request merged**